base/standard/src/main/java/org/openscience/cdk/tools/HOSECodeGenerator.java

```diff
@@ -24,6 +24,17 @@
  */
 package org.openscience.cdk.tools;
 
+import static java.util.Map.entry;
```

Comment:
@stefhk3 need to remove this, Also this would have also been reasonable rather than the new Object[][] but I am OK with that.

https://github.com/cdk/cdk/blob/d7d1c3dcd89ee3626a3dff95332988fc32d79c0d/base/standard/src/main/java/org/openscience/cdk/aromaticity/AromaticTypeModel.java#L153

https://github.com/cdk/cdk/blob/d7d1c3dcd89ee3626a3dff95332988fc32d79c0d/base/standard/src/main/java/org/openscience/cdk/aromaticity/AromaticTypeModel.java#L43-L109